site/app/models/gradeable/Gradeable.php

```diff
@@ -133,6 +134,9 @@ class Gradeable extends AbstractModel {
      * that contains filename, file path, and the file size.
      */
     private $split_pdf_files = null;
+    
+    /* idk why, but pylint doesn't like me commenting the type ):*/
```

Comment:
it's phpstan, and it's because `@property` is a phpDoc construct that we're not using properly (see https://manual.phpdoc.org/HTMLSmartyConverter/PHP/phpDocumentor/tutorial_tags.property.pkg.html). You can use the new `@prop` instead to get the same functionality:
`/** @prop @var array */`